tests/integration/delete.bats

```diff
@@ -12,24 +12,24 @@ function teardown() {
 }
 
 @test "runc delete" {
-  # run busybox detached
-  runc run -d --console-socket $CONSOLE_SOCKET test_busybox
+  runc run -d --console-socket $CONSOLE_SOCKET testbusyboxdelete
   [ "$status" -eq 0 ]
 
-  # check state
-  testcontainer test_busybox running
+  testcontainer testbusyboxdelete running
 
-  runc kill test_busybox KILL
+  runc kill testbusyboxdelete KILL
   [ "$status" -eq 0 ]
-  # wait for busybox to be in the destroyed state
-  retry 10 1 eval "__runc state test_busybox | grep -q 'stopped'"
+  retry 10 1 eval "__runc state testbusyboxdelete | grep -q 'stopped'"
 
-  # delete test_busybox
-  runc delete test_busybox
+  runc delete testbusyboxdelete
   [ "$status" -eq 0 ]
 
-  runc state test_busybox
+  runc state testbusyboxdelete
   [ "$status" -ne 0 ]
+
+  if test -n "$(find /sys/fs/cgroup -wholename '*testbusyboxdelete*')"; then
```

Comment:
nit: maybe add `-type d` and print what's found, also check find exit code (to prevent false passing).

Something like
```bash
  run find /sys/fs/cgroup -wholename '*testbusyboxdelete*' -type d
  [ "$status" -eq 0 ]
  [ "$output" = "" ] || fail "cgroup not cleaned up correctly: $output"
```

which results in
```
1..4
not ok 1 runc delete
# (from function `fail' in file tests/integration/helpers.bash, line 226,
#  in test file tests/integration/delete.bats, line 40)
#   `[ "$output" = "" ] || fail "cgroup not cleaned up correctly: $output"' failed
....
# cgroup not cleaned up correctly: /sys/fs/cgroup/perf_event/system.slice/runc-testbusyboxdelete.scope
# /sys/fs/cgroup/freezer/system.slice/runc-testbusyboxdelete.scope
# /sys/fs/cgroup/hugetlb/system.slice/runc-testbusyboxdelete.scope
# /sys/fs/cgroup/net_cls,net_prio/system.slice/runc-testbusyboxdelete.scope
# /sys/fs/cgroup/cpuset/system.slice/runc-testbusyboxdelete.scope
...
```

feel free to ignore though

Reply:
amended